README.md

```diff
@@ -61,8 +61,10 @@ The format of this glossary is modeled after [HugoGiraudel/SJSJ](https://github.
 * **ARM Cortex M**:
     family of 32-bit RISC _[ARM](#ARM-architecture)_ processors widely used in embedded systems
 * **Arduino (platform)**<a name="arduino-platform"></a>:
-    open-source electronics platform based on easy-to-use hardware and software, intended for anyone making interactive projects
-* **Arduino IDE**:
+    open-source electronics platform based on easy-to-use hardware and software, intended for anyone making interactive projects and prototyping products
+* **Adruino Compatible**:
```

Comment:
I googled arduino-compatible and everyone is grammatically incorrect, because computers :)